R/get_chemscorev1.6.71.R

```diff
@@ -1,1593 +1,955 @@
-get_chemscorev1.6.71 <-
-function(chemicalid,mchemicaldata,corthresh,global_cor,mzid,max_diff_rt=10,level_module_isop_annot,
-adduct_table,adduct_weights,filter.by=c("M+H"),max_isp=100, MplusH.abundance.ratio.check=TRUE,mass_defect_window=0.01,mass_defect_mode="pos",outlocorig)
-{
-    
-   
-    #new
-    #print("here")
-    setwd(outlocorig)
-   # load("step1_results.Rda")
-    
-  #  load("global_cor.Rda")
-    
-   
-    
-    outloc1<-paste(outlocorig,"/stage2/",sep="")
-	suppressWarnings(dir.create(outloc1))
-	setwd(outloc1)	
-
-    mchemicaldata$mz<-as.numeric(as.character(mchemicaldata$mz))
-    
-    mchemicaldata$time<-as.numeric(as.character(mchemicaldata$time))
-    mchemicaldata$MonoisotopicMass<-as.numeric(as.character(mchemicaldata$MonoisotopicMass))
-    
-    level_module_isop_annot$mz<-as.numeric(as.character(level_module_isop_annot$mz))
-    
-    level_module_isop_annot$time<-as.numeric(as.character(level_module_isop_annot$time))
-    
-    chemical_score<-(-100)
-    
-    fname<-paste(chemicalid,"data.txt",sep="_")
-    
-    
-    if(length(mchemicaldata$mz)>0){
-        
-        if(length(which(duplicated(mchemicaldata$mz)==TRUE))>0){
-          #  mchemicaldata<-mchemicaldata[-which(duplicated(mchemicaldata$mz)==TRUE),]
-        }
-        
-        mchemicaldata$Module_RTclust<-gsub(mchemicaldata$Module_RTclust,pattern="_[0-9]*",replacement="")
-        
-        
-        
-        mchemicaldata_orig<-mchemicaldata
-       
-        #if(FALSE)
-        {
-            if(length(which(mchemicaldata_orig$Adduct%in%as.character(adduct_weights[,1])))>0){
-                rel_adduct_data<-mchemicaldata[which(mchemicaldata$Adduct%in%as.character(adduct_weights[,1])),]
-                
-                rel_adduct_module<-gsub(rel_adduct_data$Module_RTclust,pattern="_[0-9]*",replacement="")
-                
-                module_rt_group<-gsub(mchemicaldata$Module_RTclust,pattern="_[0-9]*",replacement="")
-                
-                mchemicaldata<-mchemicaldata[which(module_rt_group%in%rel_adduct_module),]
-                
+#' @export
+do_something <- function(i,

```

Comment:
is this function's changed name an improvement?

Reply:
No it isn't, but I still don't really know what it does!

Reply:
Yeah, I went through it a little and then merged. It is confusing at the very least. 🤔